commands/transactions.go

```diff
@@ -0,0 +1,25 @@
+package commands
+
+import (
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/querybuilders"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+)
+
+// RecordTransaction holds the arguments required to record a user transaction.
+type RecordTransaction struct {
+	Metadata    querybuilders.Metadata `json:"metadata"`    // Metadata associated with the transaction.
+	Hex         string                 `json:"hex"`         // Hexadecimal string representation of the transaction.
+	ReferenceID string                 `json:"referenceId"` // Reference ID for the transaction.
```

Comment:
Please double check if it's `referenceId` or `referenceID` (possible mistake here) 🙏

Reply:
This is the copy of the structure used in SPV Wallet API Transactions record handler.  Based on the tag and tested behavior it is _referenceId_.